concepts/05 Widgets/Form/01 Getting Started with Form/Getting Started with Form.md

```diff
@@ -0,0 +1,3 @@
+The Form component displays fields from data objects as a collection of label-editor pairs. You can process the form data, validate it, and send to a server.
+
+This tutorial shows how to add the Form to a page, create form items, adjust them, validate items data and submit the form. As a result, you will get a widget that looks as follows:
```

Comment:
A screenshot from the final form can increase interest here
Something like https://js.devexpress.com/Documentation/Guide/Widgets/DataGrid/Getting_Started_with_DataGrid/

Reply:
This is not a screenshot, but a real working app. I'm working on it at the moment. As soon as I finish, the source code will be available here: https://github.com/DevExpress-Examples/getting-started-with-form/